install-deps.sh

```diff
@@ -66,7 +66,7 @@ function munge_debian_control {
     fi
     if $with_jaeger; then
 	sed -i -e 's/^# Jaeger[[:space:]]//g' $control
-	sed -i -e 's/^# Crimson      libyaml-cpp-dev,/d' $control
+	sed -i -e 's/^# Crimson[[:space:]]\+libyaml-cpp-dev/               libyaml-cpp-dev/g' $control
```

Comment:
@clwluvw we specifically want to use libyaml 0.6+ above, if not jaeger build will break and that's why we remove libyaml-cpp-dev(using sed to delete that line) which is fine if we use seastar(here, for crimson) instead we add 
```
Built-Using:   libyaml-cpp-dev (>= 0.6),
```
did you get my point? I would be happy to approve once you address these changes

Reply:
@ideepika sorry for the delay. PTAL